gluefactory/datasets/oxford_paris_mini_1view_jpldd.py

```diff
@@ -60,8 +66,13 @@ class OxfordParisMiniOneViewJPLDD(BaseDataset):
             },
             "line_gt": {
                 "data_keys": ["deeplsd_distance_field", "deeplsd_angle_field"],
+                "use_binary_gt": False,
+                "thickness": 1,  # binary df line thickness
                 "enforce_threshold": 5.0,  # Enforce values in distance field to be no greater than this value
             },
+            "augment": {
```

Comment:
I would like to see that we can switch augmentation on and off. One of two possibilities:

1.  add a type "none" that will not apply any augmentations
2. add a k/v pair with key "do" and Boolean type as value. Then if its activated and which type is used are separately configurable

In both cases ADD DOCUMENTATION!! -> How is the augmentation applied(additional augmented images or probability of augmentation on each image load + what options for augmentation)